jsonrpc/types.go

```diff
@@ -102,6 +102,7 @@ type block struct {
 	Hash            types.Hash          `json:"hash"`
 	Transactions    []transactionOrHash `json:"transactions"`
 	Uncles          []types.Hash        `json:"uncles"`
+	BaseFee         argUint64           `json:"baseFee"`
```

Comment:
Blocks without `London` fork enabled should not have the `BaseFee`.

Reply:
Added omitempty option